react/Provider/provider.tsx

```diff
@@ -1,9 +1,16 @@
 /* eslint-disable vtex/prefer-early-return */
 import type { FC } from 'react'
 import React, { useState } from 'react'
+import { useMutation } from 'react-apollo'
 
+import uploadFile from '../queries/uploadFile.gql'
+import saveMasterdata from '../queries/saveMasterdata.gql'
 import Context from '../Context/context'
 
+const BUTTON_CHOICE_IS_IMAGEM = 1
+const BUTTON_CHOICE_IS_TEXT = 2
+const BUTTON_CHOICE_IS_HTML = 3
+
 const Provider: FC = props => {
   const [button, setButton] = useState(1)
```

Comment:
Just missing here :)

```suggestion
  const [button, setButton] = useState(BUTTON_CHOICE_IS_IMAGEM)
```

Reply:
I believe this [suggestion ](https://github.com/vtex-apps/badges/pull/3#discussion_r675807551) is a better solution in this case 😄